packages/thumbprint-tokens/package.json

```diff
@@ -22,6 +28,7 @@
         "fs-extra": "^8.1.0",
         "glob": "^7.1.2",
         "handlebars": "^4.2.0",
+        "jest": "^24.9.0",
```

Comment:
Can you remove this (and the `test` script) since we run Jest at the root level?

Reply:
done